### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
:-)

### CODE REVIEW

**Review by Abestanis — CHANGES_REQUESTED** [review]
The new translations look very good, thanks for updating them.

There is one missing translation and some warnings that were generated with the new tool in #410 which I would like to fix. All of the warnings are in the `main_nl.ts`.
* In line `290`, the translation ends with an `!` whereas the source ends in `.`. In my opinien, the exclamation mark is totally justified because it is a warning. if you agree, please add the following comment to suppress the warning in the future:
```xml
        <translatorcomment>checkTranslation ignore: punctuation_end_differ</translatorcomment>
```
* In line `392`, the translation ends with a `;` instead of a `,`. I think that was probably also a choice you made, and not a mistake. If so, please also a the above comment here.
* In line `416`, the `11` from the source does not occur in the translation. I talked about this one in #410, if that is not a mistake and you just rephrased the sentence, please add the appropriate comment.
* In line `440` the translation is missing.
* In line `704`, the translation ends with a `:` instead of a `...`. Again, I believe this was a decision you made, so please add the `punctuation_end_differ` comment.
* In line `833`, the last translation, the source ends with a `.` and the translation does not. Please don't change it, because I think you are totally right, there should be no dot. Unfortunately, I cant change the source because we are in the time frame before the release where we promised not to change the translations because it makes your work more difficult. But I will make sure to remove that dot after the release.

Sorry for taking more of your time and thanks again for updating it.

File: src/translations/main_nl.ts [hunk 1]
```diff
@@ -433,23 +433,23 @@ Mogelijk is OpenSSL niet geïnstalleerd.</translation>
     </message>
     <message>
         <source>Unread change cmd:</source>
-        <translation type="unfinished"></translation>
+        <translation>Ongelezen wijziging-opdracht:</translation>
     </message>
     <message>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If not empty, this command will be invoked every time the unread counter changes (including when it becomes zero). It is invoked via shell, as-is, with %NEW% replaced by the new unread count value, and %OLD% replaced with the old unread count value (which may be the same as new).&lt;/p&gt;&lt;p&gt;Most users don&apos;t need this functionality and should leave it empty.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
         <translation></translation>
```
[the remark this answers is not in the document]
Comment (Vistaus) [reply src/translations/main_nl.ts:440]: No worries, I'm glad to help out, even if it takes some more time :)

A few things are Qt Linguist's fault as it didn't show those things. But I think I fixed all of the issues now (I added comments to a couple of them, as you suggested, and fixed the other ones per your instructions).

**Comment by Vistaus** [comment]
No worries, I'm glad to help out, even if it takes some more time :)

A few things are Qt Linguist's fault as it didn't show those things. But I think I fixed all of the issues now (I added comments to a couple of them, as you suggested, and fixed the other ones per your instructions).

File: src/translations/main_nl.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Abestanis) [reply src/translations/main_nl.ts]: I was talking about the empty translation in line 440, but I see you added that.

File: src/translations/main_nl.ts [hunk 2]
```diff
@@ -703,6 +705,7 @@ Het logboek is opgeslagen als %2</translation>
         <source>Snooze for ...</source>
         <translation>Onderdrukken:</translation>
     </message>
+        <translatorcomment>checkTranslation ignore: punctuation_end_differ</translatorcomment>
```
Comment (Abestanis) [inline src/translations/main_nl.ts:708]:
```suggestion
        <translation>Onderdrukken:</translation>
        <translatorcomment>checkTranslation ignore: punctuation_end_differ</translatorcomment>
    </message>
```
Unfortunately, this `translatorcomment` is in the wrong spot.

File: src/translations/main_nl.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vistaus) [reply src/translations/main_nl.ts]: Ah, I understand.

**Review by Abestanis — APPROVED** [review]
Awesome, thanks for the update!

**Comment by Vistaus** [comment]
> Awesome, thanks for the update!

No problem! 😃 👍

**Pull request merged**